src/main/java/com/elixr/poc/rest/response/PurchaseGetResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.elixr.poc.rest.response;
+
+import com.elixr.poc.data.Purchase;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Send successful response to user while reterival operations.
+ */
+@Data
+public class PurchaseGetResponse extends ErrorResponse {
```

Comment:
Where is PurchaseGetResponse used?

Reply:
OK, so by mistake, i Commit this PurchaseGetResponse, Actually we need this in some othe API.

Reply:
Updated